### ISSUE TITLE
DSUs do not have beauty based on their quality

### ISSUE THREAD

**Issue by lilwhitemouse** [lilwhitemouse/RimWorld-LWM.DeepStorage#33]
One perhaps helpful comment:

The difference I see is that there is a <statBases> for Beauty in the dresser and not in the shelving. Perhaps include a <Beauty> field.

**Comment by Dyrewulfe** [comment]
Hello. I noticed this myself and came to post an issue about it, but I see you're aware of it!

Do they really need to have a quality level, anyways? It's a beef of mine with the vanilla game that certain silly things have a functionally useless quality level(I'm looking at you, vanilla shelf!). It just acts as pointless wealth inflation(and creates incentive to make intentionally bad furniture, if one is that kind of player.)

Could perhaps just change the parent def from FurnitureWithQualityBase to FurnitureBase? This is what I did when got sick of having 50+ masterwork shelves/baskets/etc inflating my colony value and still being ugly.
